test/grpc/streaming/write_last_after_reply_two_requests.cc

```diff
@@ -0,0 +1,45 @@
+#include "streaming.h"
```

Comment:
Let's make this explicit like we do (or should be doing) in other files. Here and all the other `*.cc` files in this PR please!
```suggestion
#include "test/grpc/streaming/streaming.h"
```